rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -751,7 +744,15 @@ error_code cellGameGetParamInt(s32 id, vm::ptr<s32> value)
 	}
 	}
 
-	*value = psf::get_integer(prm->sfo, key, 0);
+	s32 v = psf::get_integer(prm->sfo, key, 0x12345678);
```

Comment:
sfo is just a std::map... you don't need that hack to check whether it has an element.